pypeit/spectrographs/vlt_fors.py

```diff
@@ -336,9 +336,12 @@ def config_specific_par(self, scifile, inp_par=None):
             par['flexure']['spec_method'] = 'skip'
             #par['reduce']['skysub']['bspline_spacing'] = 0.6
 
-        if 'lSlit' in self.get_meta_value(scifile, 'decker') or 'LSS' in self.get_meta_value(scifile, 'decker'):
-            par['calibrations']['slitedges']['sync_predict'] = 'nearest'
-
+        try:
+            if 'lSlit' in self.get_meta_value(scifile, 'decker') or 'LSS' in self.get_meta_value(scifile, 'decker'):
+                par['calibrations']['slitedges']['sync_predict'] = 'nearest'
+        except:
+            # if the above fails, we are probably in pypeit_sensfunc
+            msgs.info('Decker not detected; if you are running pypeit_sensfunc, do not panic!')
```

Comment:
My recollection was that we always tried to include the original file header in the pypeit output products.  Is that not true for FORS2?  It could be that `get_meta_value` is looking at the header of the wrong extension for the `spec1d` files.

I would isolate the specific line that we know is causing the problem (it also saves us from having to open the file twice to read one value), so I would do the following instead:
```
try:
    decker = self.get_meta_value(scifile, 'decker')
except TypeError as e:
    msgs.warn('Decker not detected; if you are running pypeit_sensfunc, do not panic!')
else:
    if any([t in decker for t in ['lSlit', 'LSS']]):
        par['calibrations']['slitedges']['sync_predict'] = 'nearest'
```